### DISCUSSION TITLE
Welcome to mercator Discussions!

### DISCUSSION THREAD

**Discussion by dbarzin** [dbarzin/mercator#93] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by etienne-64** [comment]
hello, 

my name is Stéphane and I have been working for many years in the world of SMEs and information technology. I am interested in all topics that combine entrepreneurship and technology, strategical and operational aspects. Currently, I'm a CISO for a SME in the South West of France.

Concerning coding, I developed in J2EE, in PHP/Symfony, in Python/Flask but I have been coding fewer for the last few years.

Why I join Mercator project:
- the desire to contribute to an opensource project,
- the desire to contribute to a useful project for SMEs that suffer a lack of tools,
- the desire to code again.

I like exchange with other people and I hope that this remote project will lead us to meet in real life for this project or another ;)

Stephane
